fluentd/configs.d/openshift/output-es-config.conf

```diff
@@ -3,14 +3,16 @@
       host "#{ENV['ES_HOST']}"
       port "#{ENV['ES_PORT']}"
       scheme https
-      index_name ${record['kubernetes_namespace_name']}.${record['kubernetes_namespace_id']}.${Time.at(time).getutc.strftime(@logstash_dateformat)}
+      index_name project.${record['kubernetes']['namespace_name']}.${record['kubernetes']['namespace_id']}.${Time.parse(record['@timestamp']).getutc.strftime(@logstash_dateformat)}
       user fluentd
       password changeme
 
       client_key "#{ENV['ES_CLIENT_KEY']}"
       client_cert "#{ENV['ES_CLIENT_CERT']}"
       ca_file "#{ENV['ES_CA']}"
 
+      type_name com.redhat.viaq.common
```

Comment:
Like this?
-       queryCommand := `oc exec ` + kibana_pod + ` -- curl [...] "https://` + es_svc + `/` + index + `.*/fluentd/_search?q=hostname:` + hostname + `&fields=message&size=` + querySize + `"`
+       queryCommand := `oc exec ` + kibana_pod + ` -- curl [...] "https://` + es_svc + `/` + index + `.*/com.redhat.viaq.common/_search?q=hostname:` + hostname + `&fields=message&size=` + querySize + `"`